### PR TITLE
README.md: Add step for downloading binaries

### PR DESCRIPTION
**Pull request by sedflix**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,6 +71,10 @@ higher-level APIs:
    ```
    kind create cluster
    ```
+1. Download required binaries:
+  ```
+  ./scripts/download-binaries.sh
+  ```
```
Comment (makkes) [inline README.md:77]:
```suggestion
1. Download required binaries:
  This step is only necessary if you don't have `kubectl` and `helm` on your PATH!
  ```
  ./scripts/download-binaries.sh
  export PATH=$(pwd)/bin:${PATH}
  ```
```

**Review by makkes — COMMENTED** [review]
I added a minor suggestion. wdyt?

File: README.md (hunk 1 above)
Comment (hectorj2f) [reply README.md:77]: +1

Comment (xunpan) [reply README.md:77]: I'd like to have both `./scripts/download-e2e-binaries.sh` and `./scripts/download-binaries.sh` in the step.
The `download-e2e-binaries.sh` downloads `kind`.

**Comment by makkes** [comment]
@sedflix this PR is stale but if you'd like to see this merged, don't hesitate to respond to the questions above, please. Thanks!

**Pull request closed without merging**